src/chainparams.cpp

```diff
@@ -102,6 +102,13 @@ class CMainParams : public CChainParams {
         consensus.nPowMaxAdjustUp = 16; // 16% adjustment up
         consensus.nPreBlossomPowTargetSpacing = 45;
         consensus.nPostBlossomPowTargetSpacing = 45;
+        consensus.rewardSteps[0] = 300000;
```

Comment:
a comment explaining your choice of magic numbers wbn -- like if these correlate to expected years or something?

Reply:
should probably be bumped up since that height is almost upon us, but 174720 is the number of blocks in a quarter (3 months)